1-the-basics/style.css

```diff
@@ -0,0 +1,27 @@
+header, footer{
+    background-color: limegreen;
+}
+form{
+    background-image: url(https://s1.1zoom.me/b5050/192/Lapland_region_Finland_510265_3840x2400.jpg);
+    background-size: cover;
+    background-color: whitesmoke;
```

Comment:
```suggestion
    background-color: whitesmoke;
    background-image: url(https://s1.1zoom.me/b5050/192/Lapland_region_Finland_510265_3840x2400.jpg);
    background-size: cover;
```

Reply:
- Set a background-image to the **element that contain**s the `form`.
- Change the background-color of the `form`.

In that way, you will see both of the rules.